### PR TITLE
first try

### PR DESCRIPTION
**Pull request by Fugas252mm**
[DEMO LINK](https://fugas252mm.github.io/react_tabs-js/)

### CODE REVIEW

File: src/components/Tabs/Tabs.jsx [hunk 1]
```diff
@@ -1 +1,40 @@
...
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={tab.id === activeTab.id ? 'is-active' : ''}
```
Comment (volodymyr-soltys97) [inline src/components/Tabs/Tabs.jsx:21]: Use the classnames library for add classes with condition

File: src/components/Tabs/Tabs.jsx [hunk 2]
```diff
@@ -1 +1,40 @@
...
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={handleTabClick(tab.id)}
+              >
+                {tab.title}
```
Comment (volodymyr-soltys97) [inline src/components/Tabs/Tabs.jsx:28]: Use destructuring for `tab`

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, there is no demo link, also, you need to add demo link to the PR description 
image

**Comment by Fugas252mm** [comment]
[DEMO LINK](https://fugas252mm.github.io/react_tabs-js/)

File: src/App.jsx [hunk 3]
```diff
@@ -1,44 +1,32 @@
...
+  const handleTabSelected = tabId => {
+    setActiveTabId(tabId);
+  };
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
```
Comment (GUSILLUS) [inline src/App.jsx:20]: Consider removing this variable and changing your state to take not only id, but a full tab

File: src/components/Tabs/Tabs.jsx [hunk 4]
```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
```
Comment (GUSILLUS) [inline src/components/Tabs/Tabs.jsx:5]: If you have done previous comment you can remove this also

File: src/components/Tabs/Tabs.jsx [hunk 5]
```diff
@@ -1 +1,37 @@
...
+              <a href={`#${id}`} data-cy="TabLink" onClick={handleTabClick(id)}>
+                {title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {activeTab.content}
```
Comment (GUSILLUS) [inline src/components/Tabs/Tabs.jsx:33]: `activeTab.content` => `content`

File: src/components/Tabs/Tabs.jsx [hunk 6]
```diff
@@ -1 +1,37 @@
...
+    if (tabId !== activeTab.id) {
+      onTabSelected(tabId);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(({ id, title }) => (
```
Comment (GUSILLUS) [inline src/components/Tabs/Tabs.jsx:18]:
```suggestion
          {tabs.map(({ id, title, content }) => (
```

**Review by GUSILLUS — CHANGES_REQUESTED** [review]
GJ!

**Pull request closed without merging**